### PR TITLE
Layout fixes

### LINKED ISSUE #3: Unable to localize results descriptions
We should have a way of localizing "Locality in France" or "Venue in France" label under each result title IMO.
See screenshot for an example.

image

### PR DESCRIPTION
**Pull request by ianthetechie**
This fixes a laundry list of layout issues, and brings a few visual enhancements:

* The `name` field is displayed rather than the full `label`. This is more appropriate for the widget as it is currently sized (even on desktop). This applies to the text box as well, which would overflow on selection of nearly all results in certain localization combinations.
* The subtitle has been reworked to show a short but relevant geographic context dynamically based on the type of search result.
* The layer has been removed in favor of iconography. While it does lose some information, I think that a typical end user cares nothing for the differences between a localadmin and a locality. I evaluated a few sets including FontAwesome, but settled on Google's Material icons as they had the best range for our use case. This also incidentally means that 100% of the result content is automatically localized now!
* Closes #3 (I had actually forgotten there was an issue for this, but it's been nagging me for ages 😱)

# Screenshots

## Before

South Korea Kyeongki-Do Gimpo

Example of label truncation. Before, this was completely unusable in South Korea with an English locale preference.

Powered by Stadia Maps

Jarring partial localization

## After

• Geolpojungang Park

* Text no longer truncates
* Location pin icon for venues (basically _any_ imaginable type of POI so we have to go with something fairly generic)
* Much more useful contextual info, without taking too much space

Rataskaevu 16
WARRAP, SOUTH SUDAN
SYLVAN LAKE, UNITED STATES
1754 sylvan

Examples of results from other layers.

Powered by Stadia Maps

And finally, no more jarring half-localization within the result cells.

### CODE REVIEW

File: src/123.svg
Comment (lseelenbinder) [inline src/123.svg:1]: Let's move all images to their own directory. It feels a bit odd to have `svg`s living next to `.ts` files.

File: .stylelintrc [hunk 1]
```diff
@@ -1,9 +1,8 @@
 {
```
Comment (lseelenbinder) [inline .stylelintrc:1]: I'm guessing this and all the other lint-related changes are due to an upgrade in eslint? It _appears_ we're dropping prettier for styles—is there a reason for that?

File: README.md [hunk 2]
```diff
@@ -22,9 +17,19 @@ Adding the search box to a map is straightforward:
 2. Instantiate the control (optionally customize the settings).
 3. Add the control to the map.
 
-### Using Modules
+### Getting a Stadia Maps account
```
Comment (lseelenbinder) [inline README.md:20]:
```suggestion
### Create a Stadia Maps account
```

**Review by lseelenbinder — CHANGES_REQUESTED** [review]

File: src/123.svg
Comment (ianthetechie) [reply src/123.svg:1]: Yeah will do. And move the logo too.

File: .stylelintrc (hunk 1 above)
Comment (ianthetechie) [reply .stylelintrc:1]: > I'm guessing this and all the other lint-related changes are due to an upgrade in eslint?

Yes, and prettier and stylelint.

> It appears we're dropping prettier for styles—is there a reason for that?

Some lint in stylelint after upgrading complained about the way we were doing it being deprecated and causing conflicts and it appears the defaults are a lot better now, so I just removed it.

If you have the patience to figure out what's needed to bring back the previous setup, then go for it, but it didn't feel worth it to me after already spending 2x the amount of time on package breakage (mostly relating to linters and formatters) than I spent actually improving the code :)

**Pull request merged**